### ISSUE TITLE
Finch

### ISSUE THREAD

**Issue by oodbur** [bascht/omemo-top#57]
it's said to work somehow:
https://github.com/gkdr/lurch#does-it-work-in-finch

what are the criteria for the progress states? i guess the implementation status should be set to at least "testing: yes".

**Comment by bascht** [comment]
@oodbur Yup _Mostly, but I only tried it briefly._ sounds like we could set `testing: yes`. Do you want to submit a PR?

**Comment by oodbur** [comment]
if that's necessary...

**Comment by oodbur** [comment]
i don't see clear criteria for these statuses. Gajim and Pidgin are set to "done" while i would not recommend them to regular users given their serious security or useability flaws and they are arguably not more advanced than Finch's plugin, which is the exact same thing as the Pidgin plugin.

**Comment by bascht** [comment]
I think _arguably_ is the correct keyword. Gajim and Pidgin are marked `done` because you can just go ahead and install the respective OMEMO plugin. If this applies to Finch as well, I have no problem setting Finch to `done`. :)

**Comment by oodbur** [comment]
sounds like a useful criterium.
but for most (potential) users i'm dealing with that would exclude Pidgin/lurch atm since you still have to compile yourself.

**Comment by oodbur** [comment]
so, maybe i've just completed `testing` Finch:
installing lurch for Finch is the very same as installing it for Pidgin since you actually don't install it for Finch or Pidgin but for Purple. after that can be used in either one.

**Comment by Echolon** [comment]
@oodbur push
